portal/app/core/templates/partials/main_farm.html

```diff
@@ -221,14 +266,4 @@ <h4 class="font-weight-semibold mb-0">60%</h4>
 {% endblock dashboard-main %}
 
 {% block  dashboard-scripts %}
-    <script type="text/javascript">
-        let token = '{{ token }}';
-        let homeList = {{ homes | safe }};
-        let deviceList = {{ devices | safe }};
-
-        window.localStorage.setItem('token', token);
-        window.localStorage.setItem('homes', JSON.stringify(homeList));
-        window.localStorage.setItem('devices', JSON.stringify(deviceList));
-    </script>
-    <script src="{% static "js/dashboard.js" %}"></script>
```

Comment:
These scripts aren't needed here anymore? How come?
Also, if we don't need them then we can remove the `{% block dashboard-scripts %}`

Reply:
This script is not being used currently, but we will probably link the data up soon so no need to delete this script. I moved it to dashboard_farm file instead, since that acts as a template for main_farm.